src/live/src/user-to-user/live-service-facade.js

```diff
@@ -0,0 +1,58 @@
+import PubNub from 'pubnub';
+
+import Client from '../../../client';
+import { User } from '../../../entity';
+import { PubNubListener } from './pubnub-listener';
+import { getLiveService } from './live-service';
+
+
+function _getLiveService() {
+  const client = Client.sharedInstance();
+  return getLiveService(client);
+}
+
+function register() {
+  const liveService = _getLiveService();
+  const activeUser = User.getActiveUser();
+
+  return liveService.registerUser(activeUser)
+    .then((pubnubConfig) => {
+      const pubnubClient = new PubNub(pubnubConfig);
+      const listener = new PubNubListener();
+      liveService.initialize(pubnubClient, listener);
+    });
+}
+
+function unregister() {
+  const liveService = _getLiveService();
+  return liveService.unregisterUser()
+    .then((resp) => {
+      liveService.uninitialize();
+      return resp;
+    });
+}
+
+function onConnectionStatusUpdates(func) {
+  _getLiveService().onConnectionStatusUpdates(func);
+}
+
+function offConnectionStatusUpdates(func) {
+  _getLiveService().offConnectionStatusUpdates(func);
+}
+
+function unsubscribeFromAll() {
+  _getLiveService().unsubscribeFromAll();
+}
+
+function isInitialized() {
```

Comment:
One might need to know the current status of the `LiveService`. Perhaps after encountering an error during subscribe()/unsubscribe()... I think it's okay to expose it. Why not?